t/mojo/certs/selfsigned.crt

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
There should be instructions in `t/mojo/ioloop_tls.t` for how to regenerate all certs. But are you sure the existing certs don't already cover this case?